LondonBritishLibrary/orient/BLorient765.xml

```diff
@@ -104,13 +73,137 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+</msItem>
+
+</msContents>
+
+<physDesc>
+    <objectDesc form="Leaf">
+        <supportDesc>
+            <support>
+                <material key="parchment"/>
+            </support>
+            <extent>
+                <measure unit="leaf">1</measure>
+            </extent>
+        </supportDesc>
+
+        <layoutDesc>
+            <layout columns="2" writtenLines="25" cert="high"/>
+        </layoutDesc>
+    </objectDesc>
+
+    <handDesc>
+
+        <handNote script="Ethiopic" xml:id="h1">
+            <desc>Good hand of the <date>18th century</date>
+            </desc>
+          <date notBefore="1700" notAfter="1800" resp="PRS10171Wright"/>
+            </handNote>
+
+    </handDesc>
+    <additions>
+
+<list>
+
+<item xml:id="e1">
+<desc type="MixedNote">The name of the first owner has been erased, and that of <foreign xml:lang="gez">ክንፈ፡ ሚካኤል፡</foreign>substituted.</desc>
+</item>
+
+</list>
+
+    </additions>
+
+</physDesc>
+
+
+</msPart>
+
+<msPart xml:id="p2">
+<msIdentifier></msIdentifier>
+<msContents>
+
+<msItem xml:id="p2_i1">
+<locus target="3r"/>
+<title type="complete" ref="LIT2169Pateri"/>
+</msItem>
+
+</msContents>
+
+<physDesc>
+    <objectDesc>
+        <supportDesc>
+            <support>
+                <material key="parchment"/>
+            </support>
+        </supportDesc>
+        <layoutDesc>
+            <layout columns="2" writtenLines="18 20" cert="high"/>
+        </layoutDesc>
+    </objectDesc>
+
+    <handDesc>
+
+        <handNote script="Ethiopic" xml:id="h2">
+            <desc>Well written of the earlier part of <date>18th century</date>
+            </desc>
+          <date notBefore="1700" notAfter="1730" resp="PRS10171Wright"/>
+            </handNote>
+
+    </handDesc>
+    <additions>
+
+<list>
+
+<item xml:id="a1">
+  <locus target="#1r"></locus>
+    <desc type="OwnershipNote">The note states that the manuscript belongs to the church of
+            <placeName ref="INS0101MadhaneAlam"/>.</desc>
+        <q xml:lang="gez">ዜና፡ አበው፡ ዘ<placeName ref="INS0101MadhaneAlam"> ቅዱስ፡ መድኃኔ፡ ዓለም።</placeName></q>
+</item>
+
+<item xml:id="a2">
+<locus target="#155v"></locus>
+<desc type="GuestText"><title ref="LIT3027RepCh307">A hymn to the Christian church</title></desc>
+<q xml:lang="en"><seg part="I">ኃይልኪ፡ ተዓውቀ፡ ወበርኅበ፡ ዓለም፡ ሰፈነ። በመዋዕለ፡ ሕይወት፡ ወበወርኃ፡ ሞት፡ እስመ፡ አድምዓ፡ ሥልጣነ።</seg></q>
+</item>
+
+<item xml:id="a3">
+<locus target="#155v"></locus>
+<locus target="#156"></locus>
+<locus target="#157"></locus>
+<desc type="MixedNote">Notes in Amharic from the reigns of
+  <persName ref="PRS3692egwalas">Gʷālu</persName>, <persName ref="PRS5637IyoasI">ʾIyoʾās</persName>, <persName ref="PRS5619IyasuIV">ʾIyāsu IV</persName> and <persName ref="PRS9430Tewodros">Theodore</persName>.</desc>
+<q xml:lang="am"/>
+</item>
+
+<item xml:id="e2">
+<desc type="MixedNote">The name of the first owner <foreign xml:lang="gez">አ<gap reason="lost"/>ተ፡ ክርስቶስ፡</foreign> <locus target="#38r"/> has been
```

Comment:
no type, and the names of the owners should be encoded as <persName> with role. I think, you have already done the entries...

Reply:
Done.